virt/monitoring/virt-running-cluster-checkups.adoc

```diff
@@ -22,28 +25,28 @@ include::snippets/technology-preview.adoc[]
 
 include::modules/virt-about-cluster-checkup-framework.adoc[leveloffset=+1]
 
-== Running cluster checkups in the web console
+== Running checkups in the web console
 
-Use the web console to run a latency or storage checkup on a cluster.
-
-Use the following procedures the first time you run a latency checkup and storage checkup in the web console. For additional checkups, click *Run checkup* on either checkup tab, and select the appropriate checkup from the drop down menu.
-
-[IMPORTANT]
-====
-Before you run a latency checkup, you must first xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[create a bridge interface] on the cluster nodes to connect the VM's secondary interface to any interface on the node. If you do not create a bridge interface, the VMs will not start and the job will fail.
-====
+Use the following procedures the first time you run checkups in the web console. For additional checkups, click *Run checkup* on either checkup tab, and select the appropriate checkup from the drop down menu.
 
 include::modules/virt-latency-checkup-web-console.adoc[leveloffset=+2]
 
 include::modules/virt-storage-checkup-web-console.adoc[leveloffset=+2]
 
-include::modules/virt-measuring-latency-vm-secondary-network.adoc[leveloffset=+1]
 
-include::modules/virt-checking-cluster-dpdk-readiness.adoc[leveloffset=+1]
+== Running checkups on the CLI
 
-include::modules/virt-dpdk-config-map-parameters.adoc[leveloffset=+2]
+Use the following procedures the first time you run checkups on the CLI.
 
-include::modules/virt-building-vm-containerdisk-image.adoc[leveloffset=+2]
+include::modules/virt-measuring-latency-vm-secondary-network.adoc[leveloffset=+2]
+
+include::modules/virt-checking-storage-configuration.adoc[leveloffset=+2]
+
+include::modules/virt-checking-cluster-dpdk-readiness.adoc[leveloffset=+2]
+
+include::modules/virt-dpdk-config-map-parameters.adoc[leveloffset=+3]
+
+include::modules/virt-building-vm-containerdisk-image.adoc[leveloffset=+3]
```

Comment:
Is this specific section meant to have a +3 level offset? It was previously a +2. Right now, it seems like it's a subsection of "Running a DPDK checkup on the CLI".

If that's not intentional, I'd change the level offset to what it should be, whether that's a +2 or a +1. Otherwise this comment can be ignored

Reply:
If I understand correctly what you are asking, I believe the offsets are correct. The TOC looks like this:

▪ About the OpenShift Virtualization cluster checkup framework
▪ Running checkups in the web console
     ▪ Running a latency checkup in the web console
     ▪ Running a storage checkup in the web console
▪ Running checkups on the CLI
     ▪ Running a latency checkup on the CLI
     ▪ Running a storage checkup on the CLI
     ▪ Running a DPDK checkup on the CLI
▪ Additional resources